catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -64,6 +64,7 @@
 	public static final String MEDIA_FILE_PATH = "media_file_path";
 	public static final String CALLING_ACTIVITY = "calling_activity";
 	private static final String FILENAME_TAG = "fname=";
+	private ProgressDialog progressCircle;
```

Comment:
Please rename. `loadingDialog`, `webpageLoadingDialog`, `webviewLoadingDialog` or something better you come up with. The name should indicate that it's the loading dialog for webpages.